validators/RequestSchema.js

```diff
@@ -271,6 +271,48 @@ const UpdateSettingsRequestSchema = Joi.object().keys({
   logo: Joi.string().allow(""),
 })
 
+// Do not add validation rules for human-entered fields that cannot be checked in FormSG.
+// We don't want backend requests to fail if form submission succeeds,
+// as there is no easy way for form submitters to recover from failed requests.
+const SysCreateSiteSchema = Joi.object().keys({
```

Comment:
Do we expect these fields to be available on creation? Quite a few of these seem to be things that can only be added e.g. after site launch